src/shared/debug/launchConfiguration.ts

```diff
@@ -71,6 +71,13 @@ export class LaunchConfiguration {
             .value()
     }
 
+    /**
+     * Adds a collection of debug configurations to the top of the list
+     */
+    public async addDebugConfigurations(debugConfigs: vscode.DebugConfiguration[]): Promise<void> {
```

Comment:
Can this be exercised in launchconfiguration.test.ts

Reply:
sure, added